tools/SecurityTools/CredScanSuppressions.json

```diff
@@ -0,0 +1,1097 @@
+{
+  "tool": "Credential Scanner",
+  "suppressions": [
+    {
+      "file": "\\documentation\\development-docs\\azure-powershell-developer-guide.md",
```

Comment:
should we use / instead of \\?

Reply:
this pattern is in the original file from master branch,
new added fields followed this pattern exactly.